docusaurus/docs/engineering/start-developing/boba-frontend.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 5
+---
+
+# Install BobaFrontend
+
+:::warning
+This codebase requires Node **16**. If you're using [Node Version Manager](https://github.com/nvm-sh/nvm), you can switch to version 16 by running `nvm use 16` in your terminal.
+
+You can check the current Node version by running `node -v`.
+:::
+
+## Clone project
+
+In a terminal, run the following commands:
+
+```bash showLineNumbers
+# Clone the codebase from github
+git clone https://github.com/BobaBoard/boba-frontend.git
```

Comment:
we should really figure out if people should fork first or push to the BB repos directly :P

Reply:
Oh, yeah, this should *definitely* say fork